src/cloud-api-adaptor/test/provisioner/libvirt/provision_common.go

```diff
@@ -21,16 +21,17 @@ import (
 
 // LibvirtProvisioner implements the CloudProvisioner interface for Libvirt.
 type LibvirtProvisioner struct {
-	conn          *libvirt.Connect // Libvirt connection
-	network       string           // Network name
-	ssh_key_file  string           // SSH key file used to connect to Libvirt
-	storage       string           // Storage pool name
-	uri           string           // Libvirt URI
-	wd            string           // libvirt's directory path on this repository
-	volumeName    string           // Podvm volume name
-	clusterName   string           // Cluster name
-	kbs_image     string           // KBS Service OCI Image URL
-	kbs_image_tag string           // KBS Service OCI Image Tag
+	conn             *libvirt.Connect // Libvirt connection
+	containerRuntime string           // Either containerd or crio
```

Comment:
How about `name of the container runtime`? (listing values does not really describe the variable)